### DISCUSSION TITLE
Alpha transparency

### DISCUSSION THREAD

**Discussion by joaoibsilva** [google/model-viewer#2182] [Q&A]
Hi,

I have created a 3d model in blender with Alpha Blend and exported as *.GLB. When i test the model with "https://gltf-viewer.donmccurdy.com/" i can see the transparency but with model-viewer the model is opaque as you can see in the pics. Is there a workaround for this issue?
 Thanks!

![alpha1](https://user-images.githubusercontent.com/12135134/113176326-3f28c500-9244-11eb-957b-2a8f767b0aa7.jpg)
![alpha2](https://user-images.githubusercontent.com/12135134/113176331-3fc15b80-9244-11eb-9c9b-75be04498ab9.jpg)

**Comment by elalish** [comment]
This doesn't look like a transparency issue so much as a back-face culling issue. I can't tell you what's going on without looking at the glTF. Have you inspected it in our editor? https://modelviewer.dev/editor/

**Reply by joaoibsilva** [reply]
Hi elalish, yes i have inspected with the editor and same result. Transparency works if i use a base color in the material if i use an image as texture stops working but still works with https://gltf-viewer.donmccurdy.com/. You can access the GLB file here https://digimatter.pt/AR/assets/Neo1.glb

**Reply by elalish** [reply]
Does it matter if the image is jpg or png? Only png supports an alpha channel, so that would be interesting.

**Reply by joaoibsilva** [reply]
i have tried with png and jpg, does not work with any of them.

**Reply by elalish** [reply]
Wow, I just tried it in Don's viewer. That is something... I have no idea how it's doing what it's doing in Don's viewer; I certainly wouldn't call it physically-based. I'd guess it's some kind of transparency artifact, and not an error I want to duplicate. It is kind of a cool effect, but glTF is made for physically accurate rendering, so cool effects are kind of outside the scope.

**Reply by joaoibsilva** [reply]
ok, thanks for your help, i can get the same effect when i render in Blender using Eevee engine with "Show backface" enabled. Cycles does not give me the effect. Maybe i can ask the question in Stack and see...
Actually if you use the Babylon Sandbox viewer https://sandbox.babylonjs.com/ u get the same effect.
![eevee](https://user-images.githubusercontent.com/12135134/113461483-adbc7d00-9414-11eb-812b-07d583963d15.jpg)
